README.md

```diff
@@ -47,11 +47,11 @@ If you have docker installed, you can use the containerised version of the site.
 Clone the repository
 git submodule init
 git submodule update
+npm install
 docker-compose up
 
 (in another terminal)
-docker-compose exec lobby bash
```

Comment:
@cryogen : It looks like a newline accidentally got deleted here. Or was this intentional?

Reply:
Yes, that's intentional, I combined the two steps into one